src/BlazorUI/Demo/Client/Core/Pages/Components/Accordion/BitAccordionDemo.razor

```diff
@@ -139,4 +139,4 @@
     private byte controlledAccordionExpandedItem = 1;
     private bool AccordionToggleIsEnabled;
     private bool AccordionToggleIsExpanded;
-}
```

Comment:
please revert this change, do not remove this empty new line at the end of the file.